### ISSUE TITLE
Enhancement: EcmDocumentRouterRule support

### ISSUE THREAD

**Issue by rolandoldengarm** [SubPointSolutions/spmeta2#1071]
Maybe I'm missing something, but don't think there's support in SPMeta2 to configure routing rules on a Records Center.
Would be awesome to have that.

**Comment by SubPointSupport** [comment]
Correct, has not been implemented yet. Not much features for Records Center sites. We'll have a closer look and see what can be done. Thanks for raiding this up, @rolandoldengarm
